Description: Upstream changes introduced in version 5.22.1-5
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 maxima (5.22.1-5) unstable; urgency=low
 .
   * sync package changes with squeeze build
 .
 The person named in the Author field signed this changelog entry.
Author: Camm Maguire <camm@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- maxima-5.22.1.orig/interfaces/emacs/emaxima/maxima.el
+++ maxima-5.22.1/interfaces/emacs/emaxima/maxima.el
@@ -2495,7 +2495,7 @@ if completion is ambiguous."
     (define-key map "\C-c\)" 'maxima-check-parens)
     (define-key map [(control c) (control \))] 'maxima-check-form-parens)
 ;    (define-key map "\C-cC-\)" 'maxima-check-form-parens)
-    (define-key map "\177" 'backward-delete-char-untabify)
+;    (define-key map "\177" 'backward-delete-char-untabify)
     (setq maxima-mode-map map)))
 
 ;;;; Menu
@@ -3317,7 +3317,7 @@ To scroll through previous commands,
 (define-key inferior-maxima-mode-map [(meta control tab)] 
   'inferior-maxima-input-complete)
 (define-key inferior-maxima-mode-map "\e\t" 'inferior-maxima-complete)
-(define-key inferior-maxima-mode-map "\177" 'backward-delete-char-untabify)
+;(define-key inferior-maxima-mode-map "\177" 'backward-delete-char-untabify)
 (define-key inferior-maxima-mode-map "\C-c\C-k" 'maxima-stop)
 (define-key inferior-maxima-mode-map "\C-c\C-d" maxima-help-map)
 
